<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Integration Tests" tests="35" time="0.000">
  <testsuite name="App / Register / Sign Up" id="00bd3fc2-1388-458d-8fd2-d9e3aaec3f1b" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Register / Sign Up.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:1).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Login / Email Login" id="b739d96d-13d4-46f1-b99c-03cf2d49ee9a" timestamp="2024-08-01T06:42:23.387Z" tests="0" failures="0" errors="2" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
testScriptError: "undefined" is not valid JSON
---
]]></system-err>
  </testsuite>
  <testsuite name="App / Login / Forgot Password" id="df02ad12-5eb2-463a-a5fe-11b8158df27f" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 204" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Login / Forgot Password.]]>
        <![CDATA[Test description: Response status code is 204.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:3).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Login / Social Login" id="3aea2900-2c4f-4ffc-9042-f9c9fd464e5c" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Login / Social Login.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:4).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Profile / Add/Edit Profile Info" id="4012ae9d-9b51-4e43-861b-1dc36471a71d" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 204" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Profile / Add/Edit Profile Info.]]>
        <![CDATA[Test description: Response status code is 204.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:5).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Profile / Update Email" id="0260f4e0-bd66-489b-9aac-0b0920c4ead2" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Profile / Update Email.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:6).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Profile / Verify Email" id="e11ca8f5-94e0-44cf-8457-f91c82d6d94c" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 204" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Profile / Verify Email.]]>
        <![CDATA[Test description: Response status code is 204.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:7).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Profile / Change Password" id="95cb53ed-c780-4f2b-a5a5-87337f5e8f41" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 204" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Profile / Change Password.]]>
        <![CDATA[Test description: Response status code is 204.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:8).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Vehicles / Add Vehicle" id="694641d1-6f43-40d7-91c5-43cc02fedd3c" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Vehicles / Add Vehicle.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:9).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Vehicles / Edit Vehicle Info" id="e8aa1b5b-4bd1-4da3-8c77-b4f45c2f1eb7" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 204" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Vehicles / Edit Vehicle Info.]]>
        <![CDATA[Test description: Response status code is 204.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:10).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Vehicles / Validate Vehicle" id="32788c68-9a99-47ab-b95f-133d7bbbabfa" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Vehicles / Validate Vehicle.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:11).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Vehicles / Vehicle Rules" id="7e344319-a85e-4a25-8f2e-b96fce6343ef" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Vehicles / Vehicle Rules.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:12).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / My Sessions / Parking Session" id="2cf35edb-11db-42bf-a35b-5d7751a7c571" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / My Sessions / Parking Session.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:13).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / My Sessions / Parking Session" id="3035323e-2978-4f62-bf4d-7f6b278e5a23" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / My Sessions / Parking Session.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:14).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / My Sessions / Parking Session v2" id="f8a3ffee-40fc-47fb-a685-e405ca58f0a5" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / My Sessions / Parking Session v2.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:15).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / My Sessions / Edit Parking Session" id="cdf2c19d-aaec-49f8-8351-58e18675eca4" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 204" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / My Sessions / Edit Parking Session.]]>
        <![CDATA[Test description: Response status code is 204.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:16).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / My Sessions / Active Parking Session" id="c0338a01-66a8-46ae-9f36-ed442ea9b534" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / My Sessions / Active Parking Session.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:17).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Payments / All Cards" id="b411df26-33f5-43a9-820f-4fe67d2f605e" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Payments / All Cards.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:18).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Payments / Add Card" id="d37df9a1-2c2f-440d-b889-be26f3335909" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Payments / Add Card.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:19).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Favorites / Add Favourite Parking" id="e5011b14-68bb-4947-bdbc-8c46444132b5" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Favorites / Add Favourite Parking.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:20).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Favorites / Fetch Favorite Parking" id="138efe09-0d00-4f7f-8a9f-fdafb9c49900" timestamp="2024-08-01T06:42:23.387Z" tests="0" failures="0" errors="2" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
testScriptError: "undefined" is not valid JSON
---
]]></system-err>
  </testsuite>
  <testsuite name="App / Favorites / Remove Favourite Parking" id="3134c535-2215-47b1-b989-0c735819863b" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 204" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Favorites / Remove Favourite Parking.]]>
        <![CDATA[Test description: Response status code is 204.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:22).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Promotions/Get Free Parking / Generate Referral Code" id="a0da874d-250f-47c0-b030-7468e45b27b3" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Promotions/Get Free Parking / Generate Referral Code.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:23).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / OTP / Generate OTP" id="20f1076e-01b2-4efa-b218-b00e43d84140" timestamp="2024-08-01T06:42:23.387Z" tests="0" failures="0" errors="2" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
testScriptError: "undefined" is not valid JSON
---
]]></system-err>
  </testsuite>
  <testsuite name="App / OTP / Verify OTP" id="449136c8-7aee-4ebd-949b-d608f684b07f" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / OTP / Verify OTP.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:25).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Parking / Access Type" id="1b3392c2-f80f-4ebd-9084-e90dff2c1fd9" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Parking / Access Type.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:26).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Parking / Contracted Parking" id="cb0bcb4a-1fb3-49de-9498-cbf17ec8ef31" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Parking / Contracted Parking.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:27).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Parking / Tariff List" id="662e9780-5b88-40e1-9923-7c63f8d8bc20" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Parking / Tariff List.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:28).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Parking / Tariff Breakdown" id="a6e4ee2f-0011-4594-a72a-457e611ff948" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Parking / Tariff Breakdown.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:29).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Account / Create Account" id="d9a6b455-68a3-4c58-a9e3-dac47fccc96b" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Account / Create Account.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:30).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Account / Fetch Account Details" id="87d2de93-a5c3-458f-bae2-554921a6b858" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Account / Fetch Account Details.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:31).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Account / Edit Account Details" id="8241f5e9-dca7-4ea2-aa25-956f64124407" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 204" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Account / Edit Account Details.]]>
        <![CDATA[Test description: Response status code is 204.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:32).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Account / Fetch Consumer Profile" id="8dc898e9-3010-422e-9fb7-d09c3f0310b1" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Account / Fetch Consumer Profile.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:33).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Device / Add Device Details" id="c6b23144-faab-4f27-8c07-6ec03e711353" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 200" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Device / Add Device Details.]]>
        <![CDATA[Test description: Response status code is 200.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:34).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="App / Logout / Logout" id="ec10d985-50a4-4107-8548-4c02af6d84a5" timestamp="2024-08-01T06:42:23.387Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: AggregateError [ETIMEDOUT]: 
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)

---
]]></system-err>
    <testcase name="Response status code is 204" time="0.000" classname="IntegrationTests">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 0bad55cd-7768-404f-a196-e3ae251f35c9.]]>
        <![CDATA[Collection name: Integration Tests.]]>
        <![CDATA[Request name: App / Logout / Logout.]]>
        <![CDATA[Test description: Response status code is 204.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:35).]]>
      </failure>
    </testcase>
  </testsuite>
</testsuites>